### PR TITLE
cleanup:cluster controller log missing clustername

### PR DESCRIPTION
**Pull request by xigang**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```
NONE
```

### CODE REVIEW

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Thanks~
/lgtm

File: pkg/controllers/cluster/cluster_controller.go [hunk 1]
```diff
@@ -402,7 +402,7 @@ func (c *Controller) monitorClusterHealth(ctx context.Context) (err error) {
 					c.clusterHealthMap.delete(clusterName)
 					return true, nil
 				}
-				klog.Errorf("Getting a cluster to retry updating cluster health error: %v", clusterName, err)
+				klog.Errorf("Getting a cluster %v to retry updating cluster health error: %v", clusterName, err)
```
Comment (RainbowMango) [inline pkg/controllers/cluster/cluster_controller.go:405]: @Garrybest should we optimize this log to make it easier to understand? Any comments?

By the way, I think this log level could be `Warning`, `klog.Errorf` --> `klog.Warningf`.

Comment (Garrybest) [reply pkg/controllers/cluster/cluster_controller.go:405]: I think we can delete this line. The err will return and afterwards the details will be printed.

`klog.Errorf("Update health of Cluster '%v' from Controller error: %v. Skipping.", cluster.Name, err)`

Comment (xigang) [reply pkg/controllers/cluster/cluster_controller.go:405]: I delete this log.

**Comment by xigang** [comment]
@RainbowMango @Garrybest  Redundant log has been removed.

**Review by RainbowMango — APPROVED** [review]
/approve
Thanks.

**Pull request merged**